packages/anu-vue/src/components/drawer/ADrawer.vue

```diff
@@ -69,7 +69,7 @@ useDOMScrollLock(toRef(props, 'modelValue') as Ref<boolean>)
         v-show="props.modelValue"
         ref="refMask"
         v-bind="defaultsAttrs"
-        class="a-drawer-wrapper flex fixed inset-0 bg-[hsla(var(--a-backdrop-c),var(--a-backdrop-opacity))]"
+        class="a-drawer-wrapper flex fixed inset-0 bg-[hsl(var(--a-backdrop-c) / var(--a-backdrop-opacity))]"
```

Comment:
Have you tested this kind of changes because space in a class mean two different class?

Reply:
Yeah you are right. Need to do more work to fix it properly. Will push some commits shortly.